testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -282,6 +286,23 @@ class MiniAppDisplayActivity : BaseActivity() {
             }
         }
         miniAppMessageBridge.setChatBridgeDispatcher(chatBridgeDispatcher)
+
+        // setup HostEnvironmentBridgeDispatcher
+        val hostEnvironmentBridgeDispatcher = object : HostEnvironmentBridgeDispatcher {
+
+            override fun getHostEnvironmentInfo(
+                    onSuccess: (info: HostEnvironmentInfo) -> Unit,
+                    onError: (infoError: HostEnvironmentInfoError) -> Unit) {
+                val hostEnvironmentInfo = HostEnvironmentInfo(
+                        platformVersion = "Android " + Build.VERSION.SDK_INT,
```

Comment:
I don't think that `platformVersion` should have `Android` prepended to it because we already have the platform name in another field.

Also, I think this should use the user visible version name: `Build.VERSION.RELEASE`. But maybe it would also be good to have the SDK level in a separate field?

@laubengaier Does iOS have something comparable to the SDK level on Android (i.e. an Int value representing the current OS version)? 
If not, I guess we could add an optional field which is only set on Android.

Reply:
I have used `platformVersion = Build.VERSION.RELEASE` for default implementation. Probably, Android SDK level is an optional data 🤔